syn/python/get_kge.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Parse a yosys area report and give a kGE equivalient
+
+import argparse
+
+
+def read_lib(lib_file_path):
+    with open(lib_file_path, 'r') as f:
+        lib_file = f.readlines()
+
+    cell_dict = {}
+    weighted_dict = {}
+    cell_name = ''
+    for line_idx, line in enumerate(lib_file):
+        if line.startswith('  cell ('):
+            if cell_name != '':
```

Comment:
A more "pythonic" way to say this is just `if cell_name:` (because the empty string is falsy in Python). If you go with changing the "no cell name" case to use None, this becomes `if cell_name is not None`.

Similarly, line 26 becomes `if not cell_name` or `if cell_name is None`, depending on which version you go with.

Reply:
Will do